bin/cucumber_changelog

```diff
@@ -28,6 +36,6 @@ if (!preg_match('/(?<changes>##\s+\['.preg_quote($version, '/').'\].*?)^##\s/ms'
 echo "Found changelog:\n$changes";
 
 if (getenv('GITHUB_ACTIONS')) {
-    $changes = strtr($changes, ["%" => "%25", "\r" => '%0D', "\n" => '%0A', ':' => '%3A', ',' => '%2C']);
+    $changes = strtr($changes, ['%' => '%25', "\r" => '%0D', "\n" => '%0A', ':' => '%3A', ',' => '%2C']);
```

Comment:
Could it make sense to replace this with `rawurlencode()`?

Reply:
It might, I'm not actually sure what encoding is required here for github actions output (and then the input to the create-pull-request action). I need to get back to all this as these jobs aren't actually running / working anyway so I think probably leave it for now.